src/components/TextInput/TextInput.jsx

```diff
@@ -20,6 +20,9 @@ type Props = {|
   error: boolean,
   maxLength: string,
   data: Data,
+  id?: string,
+  name?: string,
+  style?: { [key: string]: any },
```

Comment:
Why pass styles into this component directly? This is the component library the styles needed should already be included within it.

Reply:
I did this for the sake of openness, but we can discuss if we want to keep this and/or `className`.
I'm in favor of at least leaving the latter.